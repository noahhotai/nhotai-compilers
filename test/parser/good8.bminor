
x:integer = 5;
y:integer = 8;