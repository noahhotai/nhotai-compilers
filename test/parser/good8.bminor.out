IDENTIFIER     x
COLON :
INT            integer
ASSIGN         =
INT LITERAL    0
SEMICOLON      ;
IF             if
LEFT PARENTHEIS     (
INT LITERAL    0
LESS THAN      <
INT LITERAL    0
RIGHT PARENTHESIS    )
RETURN         return
INT LITERAL    0
SEMICOLON      ;
parse error: syntax error
